src/get-latest-version.js

```diff
@@ -0,0 +1,17 @@
+import semver from 'semver'
+
+export const getLatestVersion = versions =>
```

Comment:
Should we only return the latest version from the stable channel? Or take the channel as a parameter so that we show users only the latest version for the channel their app is from?

Reply:
Channels from the perspective of the DHIS2 instance and App Management App needs some more thought I think...